includes/models/model.llms.quiz.attempt.php

```diff
@@ -774,7 +774,56 @@ public function is_passing() {
 	 * @return boolean
 	 */
 	public function can_be_resumed() {
-		return 1 === (int) $this->get( 'can_be_resumed' ) && 'incomplete' === $this->get( 'status' );
+
+		$can_be_resumed = 1 === (int) $this->get( 'can_be_resumed' ) && 'incomplete' === $this->get( 'status' ) && ! $this->has_resume_attempt_time_expired();
+
+		/**
+		 * Filters the attempt resumable status.
+		 *
+		 * @since [version]
+		 *
+		 * @param bool              $can_be_resumed Whether or not the attempt can be resumed.
+		 * @param LLMS_Quiz_Attempt $attempt        The quiz attempt object.
+		 */
+		return apply_filters( 'llms_quiz_attempt_can_be_resumed', $can_be_resumed, $this );
+	}
+
+	/**
+	 * Determine if the student's resume quiz attempt time limit is expired.
+	 *
+	 * The default resume quiz attempt time limit for a student is 1 day.
+	 *
+	 * @since [version]
+	 *
+	 * @return bool
+	 */
+	public function has_resume_attempt_time_expired() {
```

Comment:
oh that you moved the method here but you could update its logic, do we care about whether or not this is the last attempt? Nope :D